packages/fxa-payments-server/src/components/AppLayout/index.scss

```diff
@@ -0,0 +1,14 @@
+// HACK: create-react-app forbids using images outside the root level of the
+// project, so let's use local copies from fxa-content-server for now
```

Comment:
I guess this should have a followup issue.

I would typically handle this with something like this in packages.json:

```js
"build:copied_resources": "mkdir -p src/components/AppLayout/images/build && for F in mozilla.svg firefox-logo.svg ; do cp ../fxa-content-server/something/$F src/components/AppLayout/images/build/ ; done",
"postinstall": "npm run build:copied_resources"
```

Reply:
I'll file a follow up... considering a couple other options, though I'm not sure how/if they'll work. I think I read that create-react-app's settings will allow Sass to grab images from something in node_modules, so a symlink to fxa-content-server might work too

Reply:
https://github.com/mozilla/fxa/issues/1393